### PR TITLE
Sql Intro 

### PR DESCRIPTION
**Pull request by danderson0851**
Got working code, but lacks a lot of functionality. 
SQL Statements are executing with hard data vs passed in parameters
New dapper product repo class is identical to original product repo class (both are using dapper to execute sql statements).

Please look over the current code in my project and give me your notes so I can start heading in a better direction from here

### CODE REVIEW

**Comment by danderson0851** [comment]
I have made changes to my ProductRepository class and DapperProductRepository class and pushed again since I initially created this pull request, so some of the things I said in the description are no longer true

File: SqlIntro/DapperProductRepository.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        /// </summary>
+        /// <param name="prod"></param>
+        public void UpdateProduct(Product prod)
+        {
+            //This is annoying and unnecessarily tedious for large objects.
+            //More on this in the future...  Nothing to do here..
+            using (var conn = new MySqlConnection(_connectionString))
+            {
+                conn.Open();
+                conn.Execute("UPDATE product SET Name = ('Updated Boogers!') WHERE ProductID = (1000)");
```
Comment (codytwinton) [inline SqlIntro/DapperProductRepository.cs]: You're not using any parameters here. You should.

File: SqlIntro/DapperProductRepository.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        /// <summary>
+        /// Inserts a new Product into the database
+        /// </summary>
+        /// <param name="prod"></param>
+        public void InsertProduct(Product prod)
+        {
+            using (var conn = new MySqlConnection(_connectionString))
+            {
+                conn.Open();
+                conn.Execute("INSERT INTO product (Name) values('New Shenanigans!')");
```
Comment (codytwinton) [inline SqlIntro/DapperProductRepository.cs]: You're not using any parameters here. You should.

File: SqlIntro/IProductRepository.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+
+namespace SqlIntro
+{
+    public interface IProductRepository
```
Comment (codytwinton) [inline SqlIntro/IProductRepository.cs]: Missing Inner and Left Joins

File: SqlIntro/Program.cs [hunk 4]
```diff
@@ -6,17 +6,37 @@ class Program
     {
         static void Main(string[] args)
         {
-            var connectionString = ""; //get connectionString format from connectionstrings.com and change to match your database
-            var repo = new ProductRepository(connectionString);
+            var connectionString = "Server=localhost; Database=adventureworks; Uid=root; Pwd=password;"; //get connectionString format from connectionstrings.com and change to match your database
```
Comment (codytwinton) [inline SqlIntro/Program.cs]: You should be using your ConnectionManager

**Comment by danderson0851** [comment]
I think I addressed all of the elements that you mentioned needed work. I am sure there are more areas it could be improved still, but it is better than what it was...

**Pull request closed without merging**